src/Orleans/Providers/ILeaseProvider.cs

```diff
@@ -111,4 +150,9 @@ public interface ILeaseProvider
         Task Release(string category, AcquiredLease[] aquiredLeases);
     }
 
+    public class ResourceCategory
```

Comment:
Make sense, I will move this out of ILeaseProvider interface. Was hesitating then